Example/MRGPagerController/MRGDemoTabStrip.m

```diff
@@ -53,7 +53,17 @@ - (instancetype)initWithFrame:(CGRect)frame {
         self.tabIndicatorColor = [UIColor yellowColor];
     }
 
+    [NSNotificationCenter.defaultCenter addObserver:self selector:@selector(contentSizeChanged) name:UIContentSizeCategoryDidChangeNotification object:nil];
+    
     return self;
 }
 
-@end
\ No newline at end of file
+- (void)dealloc {
+    [NSNotificationCenter.defaultCenter removeObserver:self name:UIContentSizeCategoryDidChangeNotification object:nil];
+}
+
+- (void)contentSizeChanged {
```

Comment:
`contentSizeCategoryDidChange` or `contentSizeCategoryChanged` would have been clearer.

I still don't understand why we need to set the font again.

Reply:
Me neither... this is pretty dumb in my opinion. When we create a font, it's valid for the current font size setting of iOS. When we change the font size setting, the font is still computed for the previous setting so we have to reload it.
Maybe I missed something and there's a better way to do this, but it's the only solution I found online.